corpus/templates/pages/impulse.html

```diff
@@ -0,0 +1,623 @@
+                </div>
+
+                <div class="col-lg-6 text-center">
+                    <div class="container">
+                    <img src="/assets/img/events/Impulse/Shreyas-Rao.jpg" alt="Pradyumna Chari" class="img-fluid img-responsive" style="width: 300px; height: 270px; object-fit: contain;"></img>
+                    </div>
+
+                    <br>
+
+                    <h4 style="color: black;"> <strong>Shreyas Rao</strong> </h4>
+                    Shreyas Rao is a final year student from the department of Electrical and Electronics Engineering, NITK. His interests lie in the fields of signal processing and machine learning. He has previously interned at Spectrum lab - IISc Bangalore and the University of Illinois Urbana-Champaign. He is also the Chair of the Signal Processing Society at the IEEE NITK Student Chapter.
+                </div>
+                </div> -->
+                <p class='text-md md:text-lg lg:text-xl'>To be revealed Soon!</h2>
+            </div>
+        </div>
+    </section>
+
+    {% comment %} Prizes Section {% endcomment %}
+    <section id="facts" 
+        class="facts bg-base-200 text-center prose px-10 lg:px-40 py-10 min-w-full">
+        <div class=" pt-10">
+            <div class="section-header">
+                <h2 class="title text-xl md:text-2xl lg:text-3xl font-extrabold py-0 my-0">Prizes</h3>
+                <p class="section-description py-0 my-0">Exciting Prizes await you!</p>
+            </div>
+        </div>
+
+        <div class="row counters">
+
+            <!-- <div class="col-md-4 text-center">
+                <span data-toggle="counter-up">5000 </span>INR
+                <p>Winner</p>
+                        </div>
+
+                <div class="col-md-4 text-center">
+                <span data-toggle="counter-up">3000</span>INR
+                <p>First runner up</p>
+                </div>
+
+                <div class="col-md-4 text-center">
+                <span data-toggle="counter-up">2000</span>INR
+                <p>Second runner up</p>
+                        </div>
+
+                <div class="col-lg-12 col-12 text-center">
+                <center>
+                    <p style="font-size: 1.5em; ">Certificates for all those who clear preliminary round!</p>
+                </center>
+            </div> -->
+
+            <!-- ---------------------- -->
+            <div class="col-lg-12 col-12 text-center">
+                <center>
+                    <!-- <p style="font-size: 1.5em; ">Prizes and goodies worth 10k!</p> -->
+                    <p style="font-size: 1.5em; ">Prizes and goodies to be announced soon !</p>
+                </center>
+            </div>
+            <!-- -------------------------------- -->
+        </div>
+    </section>
+
+    {% comment %} Schedule Section {% endcomment %}
```

Comment:
Use timeline for schedule: https://daisyui.com/components/timeline/

Reply:
Done.